conanfile.py

```diff
@@ -202,6 +216,18 @@ def _configure_cmake(self):
         cmake.definitions['wxUSE_WEBVIEW'] = self.options.webview
         cmake.definitions['wxUSE_XML'] = self.options.xml
         cmake.definitions['wxUSE_XRC'] = self.options.xrc
+        cmake.definitions['wxUSE_HELP'] = self.options.help
+        cmake.definitions['wxUSE_WXHTML_HELP'] = self.options.html_help
+        cmake.definitions['wxUSE_URL'] = self.options.protocol
+        cmake.definitions['wxUSE_PROTOCOL'] = self.options.protocol
+        cmake.definitions['wxUSE_FS_INET'] = self.options.fs_inet
+
+        for item in str(self.options.custom_enables).split(","):
```

Comment:
What is this about? What remaining options would you set via this hack? And why not adding them statically like the rest?

Reply:
In https://github.com/wxWidgets/wxWidgets/blob/master/build/cmake/options.cmake there're a huge number of options, and obvious we can't write all those statically. However, sometimes, for certain reasons it's necessary to adjust these options(for example, when we want to control the binary size).

Reply:
Putting them in statically might be a lot of work, but having stuff like this explicit might be a lot better in the long run

WDYT @SSE4?

Reply:
of-course, I'd prefer to have to have options statically listed and explicit, this way it's much more straightforward to see which options does library have.
however, it's hard to add and synchronize them all by hand.
I'd say this is acceptable compromise.